pkg/core/dao/cacheddao.go

```diff
@@ -315,6 +316,51 @@ func (cd *Cached) DeleteStorageItem(scripthash util.Uint160, key []byte) error {
 	return nil
 }
 
+// GetStorageItemsIterator returns iterator over all storage items.
+// Function returned can be called until first error.
+func (cd *Cached) GetStorageItemsIterator(hash util.Uint160, prefix []byte) (vm.StorageIteratorFunc, error) {
+	items, err := cd.DAO.GetStorageItems(hash)
```

Comment:
It'd be nice to limit these by `prefix`.

Reply:
Fixed.